pagopa/pn-frontend-e2e
# PN-9312-refactoring-su-intervals-inutili

### Issue Description
All tests take a significant amount of time to complete; a full execution of all tests takes approximately 2 hours and 40 minutes on average.

### Changes Made:
- updated timeouts by reducing them

### How to Test
Simply run a partial or full test and observe if there are any improvements in execution time.
